examples/ConsoleApplication/appsettings.json

```diff
@@ -1,4 +1,5 @@
 {
   "AppName": "Azure App Configuration Console App",
-  "Language": "English"
+  "Language": "English",
+  "endpoints": []
```

Comment:
The casing is not consistent with existing settings.

BTW, "AppConfig:Endpoints" is probably a clearer name.